lightningd/notification.c

```diff
@@ -42,7 +46,7 @@ void notify_warning(struct lightningd *ld, struct log_entry *l)
 {
 	struct jsonrpc_notification *n =
 	    jsonrpc_notification_start(NULL, notification_topics[2]);
-	json_object_start(n->stream, "warning");
+	json_object_start(n->stream, notification_topics[2]);
```

Comment:
Using `notification_topics` with magic indices is very brittle, we should use explicit strings like we do elsewhere. Otherwise changes in the order of `notification_topics` can have unintended consequences.